### DISCUSSION TITLE
Welcome to autobsi Discussions!

### DISCUSSION THREAD

**Discussion by ccrsxx** [ccrsxx/autobsi#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sulthannaf** [comment]
Salam dari banyuwangi, jika absensi lms seperti pada umumnya yang absensi selalu menunggu dosen untuk upload materi beserta absensi di lms kemungkinan agak lambat karena kadang dosen mengupload materi beserta jadwal ketika perkuliahan berlangsung, tetapi jika absensi di aplikasi sister for student (universitas jember) sangat mudah hanya membutuhkan qr dengan jadwal yang sudah sesuai, apakah bisa kakak mengajari saya untuk auto absensi terhadap aplikasi ini (sister for student)

**Reply by ccrsxx** [reply]
Wah beda bgt klo itu beda bgt mekanisme-nya. Script ini khusus buat web my best bsi aja, dan yes ini script bisa nunggu dan refresh otomatis walau kelas mulai sampe si dosennya buka absennya.

Well, klo pake cara yang deskripsi diatas yang lewat aplikasi, harus bikin project-nya dari bawah karena beda bgt.

Saya gk bisa bantu banyak karena ini harus buat dari ulang, tapi saya bisa aja mungkin bantu kasih ide gimana cara buatnya sih klo tertarik. Asal bisa lumayan di Python, bisa aja bikin itu project sih.
